src/main/kotlin/net/minecrell/pluginyml/PlatformPlugin.kt

```diff
@@ -37,6 +38,11 @@ abstract class PlatformPlugin<T : PluginDescription>(private val platformName: S
 
     protected abstract fun createExtension(project: Project): T
 
+    protected open fun createConfiguration(project: Project): Configuration? {
+        val library = project.configurations.maybeCreate("library")
+        return project.configurations.create("${platformName}Library").extendsFrom(library)
```

Comment:
You probably want 

```suggestion
        return project.configurations.create("${platformName.decapitalize()}Library").extendsFrom(library)
```

here or it will create the configurations like `BukkitLibrary` instead of `bukkitLibrary`.